mockfirestore/document.py

```diff
@@ -70,3 +73,41 @@ def collection(self, name) -> 'CollectionReference':
         if name not in document:
             set_by_path(self._data, new_path, {})
         return CollectionReference(self._data, new_path, parent=self)
+
+
+def _apply_transformations(document: Dict[str, Any], data: Dict[str, Any]) -> Dict[str, Any]:
+    """Handles special fields like INCREMENT."""
+    increments: Dict[str, int] = {}
```

Comment:
```suggestion
    increments = {}
```

This type hint breaks the Python 3.5 build, so I reckon it's safer to remove it.